chat-widget/src/common/telemetry/TelemetryConstants.ts

```diff
@@ -118,6 +118,7 @@ export enum TelemetryEvent {
     InvalidConfiguration = "InvalidConfiguration",
     SendTypingIndicatorSucceeded = "SendTypingIndicatorSucceeded",
     SendTypingIndicatorFailed = "SendTypingIndicatorFailed",
+    event = "event",
```

Comment:
I don't think we must keep parity for this event name. Feel free to change this into "WebChatEvent".

Reply:
changed to "WebChatEvent"